<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>界面显示器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>显示当前数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>520</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>数据上传</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inlet_pipe">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>inlet pipe</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>120</y>
      <width>231</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_show">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>771</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/ppt.png)url(:/new/prefix1/ppt.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="ToolTipWidget" name="temp1" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="ToolTipWidget" name="temp2" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="ToolTipWidget" name="temp2_2" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="ToolTipWidget" name="temp2_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="ToolTipWidget" name="temp2_4" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ToolTipWidget</class>
   <extends>QWidget</extends>
   <header>tooltipwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
